librz/core/cbin.c

```diff
@@ -15,6 +15,9 @@
 
 #define LOAD_BSS_MALLOC 0
 
+#define Color_GREEN_word      "\x1b[32m"
```

Comment:
By the way, ideally we should use colors from the color theme/palette instead of hardcoding these values. See `librz/core/disasm.c` as a good example how to handle that.

Reply:
I will remove this.